modules/menu/static_menu.c

```diff
@@ -605,9 +606,18 @@ static int cmd_dialdir(struct re_printf *pf, void *arg)
 			"Audio & video must not be"
 			" inactive at the same time\n";
 
+	/* with display name */
 	err = re_regex(carg->prm, str_len(carg->prm),
-		"[^ ]* audio=[^ ]* video=[^ ]*",
-		&pluri, &argdir[0], &argdir[1]);
+		"[^ \t\r\n<]*[ \t\r\n]*<[^>]+>[ \t\r\n]*"
+		"audio=[^ \t\r\n]*[ \t\r\n]*video=[^ \t\r\n]*",
+		&dname, NULL, &pluri, NULL, &argdir[0], NULL, &argdir[1]);
+	if (err) {
+		dname = pl_null;
+		err = re_regex(carg->prm, str_len(carg->prm),
+			       "[~ ]+ audio=[^ ]* video=[^ ]*",
```

Comment:
Is there a reason why the first group (`[~ ]+`) of this regex is different from the previous implementation (`[^ ]*`)?

Reply:
'~' is with `qesc` (might  mean "quote escaping"). The quotes at beging and end are removed.